rexmex/metricset.py

```diff
@@ -45,6 +45,8 @@ def filter_metrics(self, filter: List[str] = None):
         Returns:
             self: The metric set after the metrics were filtered out.
         """
+        if not filter:
```

Comment:
Is this desired functionality? Should it even be possible to call this function with no argument?

Reply:
No it is not. I will correct this.